input/system/tembo/logs.go

```diff
@@ -101,8 +101,8 @@ func setupWebsocketForServer(ctx context.Context, wg *sync.WaitGroup, globalColl
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
+		var conn *websocket.Conn
 		for {
-			var conn *websocket.Conn
 			var cancelConn context.CancelFunc
```

Comment:
Good catch - I think we also need to push up `cancelConn` to one level up (since that one should be updated when we create a new child context for a subsequent reconnect)

Reply:
Thanks, moved this out of the loop as well https://github.com/pganalyze/collector/pull/506/commits/f0e1a358874be137fad8a0ed44d7649d92781317